client/app/components/stepper/step/step.navigation.js

```diff
@@ -0,0 +1,11 @@
+export default class {
+  constructor({
+    index,
+    relativeIndex,
+    name,
+  }) {
+    this.index = index;
```

Comment:
Instead of destructuring the parameter you could use: 

```js
Object.assign(this, parameter)
```

Is that make sense for you?

Reply:
No, because I like my parameter names to be readable :)